modules/cnv-add-nic-to-vm.adoc

```diff
@@ -1,10 +1,7 @@
 // Module included in the following assemblies:
 //
-// * cnv/cnv_users_guide/cnv-edit-vms.adoc
-// * cnv/cnv_users_guide/cnv-editing-vm-template
-//
-// Establishing conditionals so content can be re-used for editing VMs
```

Comment:
Git makes it look like this comment has been removed, but it's still there in the file.